opm/parser/eclipse/Parser/createDefaultKeywordList.cpp

```diff
@@ -34,6 +42,33 @@ int main(int argc , char ** argv) {
         bool verboseLoader = false;
         bool verboseGenerator = true;
 
+        try {
+            /* sometimes the local env's locales are broken on POSIX. Boost <=
+             * 1.56 uses the std::locale("") constructor which respects user
+             * preferred locales, and might crash. If this is the case, and
+             * we're on a non-windows system (assuming POSIX), in case of an
+             * exception, set the environment to "C" and keep going.
+             *
+             * Can be removed once boost < 1.57 is no longer supported
+             */
+            std::locale( "" );
+        } catch( const std::runtime_error& ) {
+            #if !defined(WIN32)
```

Comment:
since the code in the catch() block is required anyway, would't it be better to always use this and throw away the `try..catch` as well as the output below?